OtripleS.Web.Api.Tests.Unit/Services/Foundations/Attachments/AttachmentServiceTests.Exceptions.RetrieveAll.cs

```diff
@@ -46,18 +50,22 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAttachmentsWhenSqlExcepti
         public void ShouldThrowServiceExceptionOnRetrieveAllAttachmentsWhenExceptionOccursAndLogIt()
         {
             // given
-            var exception = new Exception();
+            var Serviceexception = new Exception();
```

Comment:
@deepshri7 naming here is also incorrect. `serviceException` is the correct choice.